client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -51,7 +51,7 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = ( {
 						{ siteVertical && (
 							<div className="onboarding-block__footer">
 								<Button className="onboarding-block__question-skip" isLink>
-									{ NO__( "Don't know yet" ) } →
+									{ siteTitle ? NO__( 'Next' ) : NO__( "Don't know yet" ) } →
```

Comment:
Noting that with this change, the button still doesn't actually work (i.e. clicking it doesn't proceed to the next step).

We can merge as-is for now, but in the long run, we should ideally pass [`goToNext`](https://github.com/Automattic/wp-calypso/blob/440a149018e0f398846807f0b8419acc4fb27128/client/landing/gutenboarding/gutenboard.tsx#L64-L66) to the block and use it. (The button label can be set depending on whether `goToNext` is `undefined` or not.)